.github/workflows/workflow.yaml

```diff
@@ -95,6 +95,8 @@ jobs:
   unittests:
     timeout-minutes: 25
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
```

Comment:
so that the commentator can comment in the PR
Not sure why it passed previously; possibly the default permissions of `GITHUB_TOKEN` has changed?